auto_derby/single_mode/commands/race.py

```diff
@@ -19,10 +19,10 @@
 
 def _choose_running_style(ctx: Context, race1: Race) -> None:
     scene = PaddockScene.enter(ctx)
-    action.wait_tap_image(templates.RACE_RUNNING_STYLE_CHANGE_BUTTON)
```

Comment:
Why remove this line?

Reply:
Explained in the main PR comment. Because `PaddockScene.choose_runing_style()` will press that button.